app/components-react/windows/go-live/dual-output/DualOutputGoLiveSettings.tsx

```diff
@@ -21,31 +22,51 @@ import UserSettingsNonUltra from './UserSettingsNonUltra';
  * - Extras settings
  **/
 export default function DualOutputGoLiveSettings() {
-  const { isAdvancedMode, isLoading, isPrime, canUseOptimizedProfile } = useGoLiveSettings().extend(
-    module => {
-      const { UserService, VideoEncodingOptimizationService } = Services;
+  const {
+    isAdvancedMode,
+    isLoading,
+    isPrime,
+    canUseOptimizedProfile,
+    isRestreamEnabled,
+    hasMultiplePlatforms,
+    enabledPlatforms,
+    primaryChat,
+    setPrimaryChat,
+  } = useGoLiveSettings().extend(module => {
+    const { UserService, VideoEncodingOptimizationService } = Services;
 
-      return {
-        isPrime: UserService.views.isPrime,
+    return {
+      isPrime: UserService.views.isPrime,
 
-        // temporarily hide the checkbox until streaming and output settings
-        // are migrated to the new API
-        canUseOptimizedProfile: false,
-        // canUseOptimizedProfile:
-        //   VideoEncodingOptimizationService.state.canSeeOptimizedProfile ||
-        //   VideoEncodingOptimizationService.state.useOptimizedProfile,
-      };
-    },
-  );
+      // temporarily hide the checkbox until streaming and output settings
+      // are migrated to the new API
+      canUseOptimizedProfile: false,
+      // canUseOptimizedProfile:
+      //   VideoEncodingOptimizationService.state.canSeeOptimizedProfile ||
+      //   VideoEncodingOptimizationService.state.useOptimizedProfile,
+    };
+  });
+
+  const shouldShowPrimaryChatSwitcher = isRestreamEnabled && hasMultiplePlatforms;
+  // TODO: make sure this doesn't jank the UI
+  const leftPaneHeight = shouldShowPrimaryChatSwitcher ? '82%' : '100%';
```

Comment:
i think some sort of `calc` operation here would be more reliable, although admittedly a bit more expensive to compute

Reply:
My very limited CSS speaking, I'd have to figure out how tall the Primary Chat switcher is, where as this seems to work? Granted, it is a hardcoded value but I believe `calc` expression would also be? Open to suggestions.

Reply:
It doesn't, it's just a one element form with a divider.